musicbrainz-server/templates/default/sv-musicbrainz-server-run.erb

```diff
@@ -5,10 +5,14 @@ set -e
 cd mb_server
 umask 022
 exec \
-    env MUSICBRAINZ_USE_PROXY=1 starman -Ilib \
-        --preload-app \
-        -E deployment \
-        --listen musicbrainz-server.socket \
-        --workers <%= @variables[:nproc] || 5 %> \
-        --user www-data \
-        --group www-data
+    env MUSICBRAINZ_USE_PROXY=1 \
+        setuidgid www-data \
+            start_server --path musicbrainz-server.socket -- \
```

Comment:
Yeah, the documentation is unclear but I think it's passed through like you said. I did use this same script in front of nginx when testing locally, and it worked (just couldn't test the setuidgid part on my Mac since it needed sudo which screws up the local::lib paths).

Looks like it's through an environment variable:
https://github.com/kazuho/p5-Server-Starter/blob/5a79791fd338338545f1461f85b912a60b02e9eb/lib/Server/Starter.pm#L162
https://github.com/kazuho/Starlet/blob/7a489d9aa39afefd2f2bad07eb6d3431f05c9cbb/lib/Plack/Handler/Starlet.pm#L14